Gemfile.lock

```diff
@@ -1,40 +1,40 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (6.1.1)
-      actionpack (= 6.1.1)
-      activesupport (= 6.1.1)
+    actioncable (6.1.3.1)
+      actionpack (= 6.1.3.1)
+      activesupport (= 6.1.3.1)
       nio4r (~> 2.0)
       websocket-driver (>= 0.6.1)
-    actionmailbox (6.1.1)
-      actionpack (= 6.1.1)
-      activejob (= 6.1.1)
-      activerecord (= 6.1.1)
-      activestorage (= 6.1.1)
-      activesupport (= 6.1.1)
+    actionmailbox (6.1.3.1)
+      actionpack (= 6.1.3.1)
+      activejob (= 6.1.3.1)
+      activerecord (= 6.1.3.1)
+      activestorage (= 6.1.3.1)
+      activesupport (= 6.1.3.1)
       mail (>= 2.7.1)
-    actionmailer (6.1.1)
-      actionpack (= 6.1.1)
-      actionview (= 6.1.1)
-      activejob (= 6.1.1)
-      activesupport (= 6.1.1)
+    actionmailer (6.1.3.1)
```

Comment:
### [actionmailer](https://rubyonrails.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 6.1.1 → 6.1.3.1

[_[change-log](https://github.com/rails/rails/blob/v6.1.3.1/actionmailer/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.1.3.1/actionmailer)_]





<details>
<summary>Commits</summary>

A change of **80** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.1.1...v6.1.3.1).

These are the first 10 commits:
- (689d096) [Merge pull request #41049 from rails/revert-39321-fix_update…](https://github.com/rails/rails/commit/689d0963e0e79f5daf0d3962c8da410b501ebaec)
- (5284a16) [Merge pull request #41030 from eileencodes/fix-preventing-wr…](https://github.com/rails/rails/commit/5284a165b8e8821fcfca62dac360bd9ed7a674b4)
- (67fd5cb) [Fix failing tests introduced by #41030](https://github.com/rails/rails/commit/67fd5cb3efe6cbd31ad88261abe9a03f186811c5)
- (9f68683) [Merge pull request #41050 from alea12/update-actioncable-doc…](https://github.com/rails/rails/commit/9f68683e57adbe7aad544e3c1c2c9e264517ca66)
- (044236e) [Merge pull request #41052 from bogdanvlviv/preload_links_hea…](https://github.com/rails/rails/commit/044236e3beec448d78bdcc3dd5209c6e90a8fc50)
- (90339be) [Merge pull request #41063 from kamipo/fix_where_with_polymor…](https://github.com/rails/rails/commit/90339be70c770e2f975e099c80f0b4462ad63297)
- (9cc99d7) [Merge pull request #40704 from alpaca-tc/sandbox_multiple_da…](https://github.com/rails/rails/commit/9cc99d70c991b586ec9e74fbc391b599eba37bb8)
- (12bb9d3) [Merge pull request #41082 from kamipo/fix_update_optimistic_…](https://github.com/rails/rails/commit/12bb9d32f56883914abcd98fd72e3c68c444808d)
- (acb6631) [Merge pull request #41090 from kamipo/fix_find_by_with_custo…](https://github.com/rails/rails/commit/acb6631cd99cdfe7db356773ef74cad7cbb570ed)
- (3f4fde4) [Merge pull request #41098 from kamipo/fix_source_type_with_s…](https://github.com/rails/rails/commit/3f4fde4d9f804140be8304b524792c052e3d1024)


</details>